app/src/pages/CounterPage/containers/CounterPageContainer.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react';
+import Counter from '../components/Counter';
+import PropTypes from 'prop-types';
+
+class CounterPageContainer extends Component {
+    constructor(props) {
+
+        super(props);
+        this.state ={
+            countValue:0,
+            parity:'even',
+            countScreen:'countScreenR'
+
+
+
+        }
+    }
+    handleIncrement =() =>{
+    this.setState((state)=>{
+        const countValue = state.countValue+1;
+        let parity= ''
+        let countScreen =''
+        if(countValue%2==0) {
+            parity='even'
+            countScreen= 'countScreenR'
+        } else {
+            parity ='odd'
+            countScreen= 'countScreenB'
+        }
+        return {
+            ...state,
+            countValue,
+            parity,
+            countScreen
+        }
+    })
+    }
+
+    handleDecrement =() =>{
+        this.setState((state)=>{
+            let countValue = state.countValue-1;
+            if (countValue<0) {
+                countValue=0
+            }
+            let parity= ''
+            let countScreen ={}
+            if(countValue%2==0) {
+                parity='even'
+                countScreen= 'countScreenR'
+            } else {
+                parity ='odd'
+                countScreen= 'countScreenB'
+            }
```

Comment:
try to think how it canbe improoved. 
HINT: try to make a logic like - if the `countValue`  grater than 0 <do something> in other case - nothing